chromium_src/chrome/browser/ui/browser.cc

```diff
@@ -58,7 +58,7 @@ Browser::Browser(const CreateParams& params)
       window_(params.window),
       tab_strip_model_delegate_(new brave::BraveTabStripModelDelegate()),
       tab_strip_model_(
-          new TabStripModel(tab_strip_model_delegate_.get(),
+          new TabStripModelImpl(tab_strip_model_delegate_.get(),
```

Comment:
cc @bbondy looks like TabStripModel is now a base class. Does that help us at all?

Reply:
We had reported a related issue a while ago: https://github.com/brave/muon/issues/395

Reply:
/cc @bbondy In C65 the change was reverted back so no need to look at this any further.